### PR TITLE
Kubevirt: vm template list: edit vm templates

### PR DESCRIPTION
**Pull request by yaacov**
Add an "Edit VM Table" kebab action.

OKD

Peek 2019-07-09 17-29

### CODE REVIEW

**Comment by yaacov** [comment]
@mareklibra @vojtechszocs @suomiy @rawagner please review.

File: frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template.tsx [hunk 1]
```diff
@@ -20,7 +25,17 @@ import { dimensifyHeader, dimensifyRow } from '../../utils/table';
 import { VMTemplateLink } from './vm-template-link';
 import { VM_TEMPLATE_LABEL_PLURAL } from '../../constants/vm-templates';
 
-export const menuActions = Kebab.factory.common;
+const vmTemplateEditAction = (kind, obj) => ({
+  label: `Edit VM Template`,
+  href: `/k8s/ns/${obj.metadata.namespace}/vmtemplates/${obj.metadata.name}/yaml`,
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template.tsx]: can we use the selectors here?

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template.tsx]: sure ! thanks :+1:

**Pull request merged**